backend/api/pretix/query.py

```diff
@@ -38,7 +38,18 @@ def _get_category_for_ticket(item, categories):
     return categories.get(category_id)
 
 
-def _create_ticket_type_from_api(item, id, categories, questions, language):
+def _get_quantity_left_for_ticket(item, quotas):
+    if item["show_quota_left"] is False:
+        return None
+
+    return sum(
```

Comment:
I don't think `sum` is correct here, I think this should be `min`, because if you have ticket in 2 quotas (A and B), and quota A has 10 available and quota B only has 1 available, you can only buy it once since otherwise you would oversell quota B

Reply:
interesting! ok, will update :)

Reply:
```suggestion
    # tickets can be in multiple quotas, in that case the one that has the least amount of tickets
    # should become the source of truth for availability. See:
    # https://docs.pretix.eu/en/latest/development/concepts.html#quotas
    return min(
```